packages/client/src/components/ProductListDropDown/ProductListDropDown.component.js

```diff
@@ -15,10 +15,8 @@ const ProductListDropdown = ({ options, ...props }) => {
     </div>
   );
 };
-
 ProductListDropdown.propTypes = {
-  options: PropTypes.string.isRequired,
-  label: PropTypes.string.isRequired,
+  options: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
This change does not belong in this PR.

Reply:
I will remove. Its also from my previous Pr.

Reply:
Why is this resolv ed if it is not removed? @patel-prakashkumar

Reply:
Resolved issue